CRITICAL - 2021-12-07 10:00:35 --> The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\ci4restapi\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#3 C:\xampp\htdocs\ci4restapi\public\index.php(28): require('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2021-12-07 10:01:28 --> The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\ci4restapi\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#3 C:\xampp\htdocs\ci4restapi\public\index.php(28): require('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2021-12-07 10:02:44 --> The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\ci4restapi\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#3 C:\xampp\htdocs\ci4restapi\public\index.php(28): require('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2021-12-07 10:05:57 --> The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\ci4restapi\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#3 C:\xampp\htdocs\ci4restapi\public\index.php(28): require('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2021-12-07 03:41:35 --> max_length=20 is not a valid rule.
#0 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(285): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('max_length=20')
#1 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('nama', 'nama', 'rr', Array, Array)
#2 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\xampp\htdocs\ci4restapi\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(27): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->create()
#8 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#9 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2021-12-07 05:40:56 --> array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'array_key_exist...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci4restapi\system\HTTP\Files\FileCollection.php(120): array_key_exists('1024', Object(CodeIgniter\HTTP\Files\UploadedFile))
#2 C:\xampp\htdocs\ci4restapi\system\HTTP\Files\FileCollection.php(84): CodeIgniter\HTTP\Files\FileCollection->hasFile('foto.1024')
#3 C:\xampp\htdocs\ci4restapi\system\HTTP\IncomingRequest.php(729): CodeIgniter\HTTP\Files\FileCollection->getFileMultiple('foto.1024')
#4 C:\xampp\htdocs\ci4restapi\system\Validation\FileRules.php(85): CodeIgniter\HTTP\IncomingRequest->getFileMultiple('foto.1024')
#5 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(277): CodeIgniter\Validation\FileRules->max_size(NULL, Array, Array, NULL)
#6 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('foto', 'foto', NULL, Array, Array)
#7 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#8 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#9 C:\xampp\htdocs\ci4restapi\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#10 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#11 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(38): CodeIgniter\BaseModel->save(Array)
#12 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->create()
#13 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#14 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
CRITICAL - 2021-12-07 05:41:37 --> array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'array_key_exist...', 'C:\\xampp\\htdocs...', 120, Array)
#1 C:\xampp\htdocs\ci4restapi\system\HTTP\Files\FileCollection.php(120): array_key_exists('1024', Object(CodeIgniter\HTTP\Files\UploadedFile))
#2 C:\xampp\htdocs\ci4restapi\system\HTTP\Files\FileCollection.php(84): CodeIgniter\HTTP\Files\FileCollection->hasFile('foto.1024')
#3 C:\xampp\htdocs\ci4restapi\system\HTTP\IncomingRequest.php(729): CodeIgniter\HTTP\Files\FileCollection->getFileMultiple('foto.1024')
#4 C:\xampp\htdocs\ci4restapi\system\Validation\FileRules.php(85): CodeIgniter\HTTP\IncomingRequest->getFileMultiple('foto.1024')
#5 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(277): CodeIgniter\Validation\FileRules->max_size(NULL, Array, Array, NULL)
#6 C:\xampp\htdocs\ci4restapi\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('foto', 'foto', NULL, Array, Array)
#7 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#8 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#9 C:\xampp\htdocs\ci4restapi\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#10 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#11 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(38): CodeIgniter\BaseModel->save(Array)
#12 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->create()
#13 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#14 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
CRITICAL - 2021-12-07 07:00:46 --> There is no data to update.
#0 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(846): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ci4restapi\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Array)
#2 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(656): CodeIgniter\Model->update('41', Array)
#3 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(64): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->update('41')
#5 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-12-07 07:00:46 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php:41
Stack trace:
#0 C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\ci4restapi\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DataException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-12-07 07:02:25 --> There is no data to update.
#0 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(846): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ci4restapi\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Array)
#2 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(656): CodeIgniter\Model->update('41', Array)
#3 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(64): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->update('41')
#5 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-12-07 07:02:25 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php:41
Stack trace:
#0 C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\ci4restapi\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DataException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-12-07 07:02:44 --> There is no data to update.
#0 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(846): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ci4restapi\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Array)
#2 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(656): CodeIgniter\Model->update('41', Array)
#3 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(64): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->update('41')
#5 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-12-07 07:04:23 --> There is no data to update.
#0 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(846): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ci4restapi\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Array)
#2 C:\xampp\htdocs\ci4restapi\system\BaseModel.php(656): CodeIgniter\Model->update('43', Array)
#3 C:\xampp\htdocs\ci4restapi\app\Controllers\Pegawai.php(64): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(824): App\Controllers\Pegawai->update('43')
#5 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ci4restapi\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4restapi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-12-07 07:04:23 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php:41
Stack trace:
#0 C:\xampp\htdocs\ci4restapi\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\ci4restapi\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\ci4restapi\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DataException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
